resources/public/api/conf/1.0/headers.raml

```diff
@@ -18,3 +18,31 @@ traits:
         type: string
         required: true
         example: 'application/vnd.hmrc.1.0+json'
+  acceptJson:
+    headers:
+      Authorization:
+        description: Authorization header based on OAuth2 token
+        type: string
+        required: true
+        example: 'Bearer QWA8BdXvGvQjqf2c4Cmql81yuW4a'
+  acceptJson:
```

Comment:
Again can you remove this header as I believe we add this header

Reply:
removed